client/public/index.html

```diff
@@ -41,7 +41,7 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
-    <div id="root"></div>
+    <main id="root"></main>
```

Comment:
we aren't :) react dumps everything inside <main>. Hence, there is no logical seperation of concerns when it comes to markup. 

Either we hard code a <header> and a <footer>, or, we create a header component and a footer component
these will have to be <sections> with classes header and footer. 

I am only changing the tag name to main, so that it is semantic in nature, and not just another div

Reply:
We will have a header, footer, navigation components and that should have the `header` , `footer`, `nav` tags etc.

Reply:
Cool. And how would react render them? will it create tags adjacent the tag with the id `root`? 
or will they be children? 
If they are children, they still need to be sections? with classnames header and nav and footer?
Asking as I hve literally no idea how react adds components

Reply:
Yes they are going to be children. The `header`, `footer`, `nav` elements. See what fit's best as per your thought process. I will appreciate your judgement here.